src/devices/Display/IFourteenSegmentDisplay.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// Represents a 14-Segment display that can display multiple digits
+    /// </summary>
+    /// <remarks>
+    /// Derived from src/devices/Display/ISevenSegmentDisplay.cs
+    /// </remarks>
+    public interface IFourteenSegmentDisplay
```

Comment:
I don't think we need this now. We generally only add interfaces when we have at least two usages for it (`ISevenSegmentDisplay` is very old, so it was basically there forever)

Reply:
Removed